doc/beamline_completion.md

```diff
@@ -104,18 +104,21 @@ Here are some key differences as relating to the Geometry:
 * No ParameterMap
   - Exact solution(s) not yet apparent, but there are several options. Given that a number of parameters are linked to the instrument and not individual components, this might affect how things are done
 * No `ComponentID` by design. Infact, there may be no requirement for unique identification of `Components` at all.
+* No CSG shapes. Dataset will record information as close as possible to the on-disk Nexus Geometry format specification for meshes and cylinders
```

Comment:
Can we make this optional? The first stage could simply reuse the current shape objects, to reduce the lead time until we have something usable?

Reply:
You mean like a `variable` of shapes?